Resources/doc/reference/installation.rst

```diff
@@ -26,6 +26,8 @@ Retrieve the bundle with composer:
 .. code-block:: bash
 
     $ php composer.phar require sonata-project/media-bundle
+    $ php composer.phar require php-http/guzzle6-adapter # recommended HTTP client
+
 
 Register these bundles in your AppKernel:
```

Comment:
you require the httplug bundle. if it should be used, it needs to be instantiated too.